src/fsipy/automatedPostprocessing/postprocessing_h5py/postprocessing_common_h5py.py

```diff
@@ -0,0 +1,470 @@
+                        help="Path to the mesh file (default: <folder_path>/Mesh/mesh.h5)")
+    parser.add_argument('--save-deg', type=int, default=2,
+                        help="Specify the save_deg used during the simulation, i.e. whether the intermediate P2 nodes "
+                             "were saved. Entering save_deg=1 when the simulation was run with save_deg=2 will result "
+                             "in using only the corner nodes in postprocessing.")
+    parser.add_argument('--stride', type=int, default=1,
+                        help="Desired frequency of output data (i.e. to output every second step, use stride=2)")
+    parser.add_argument('--dt', type=float, default=0.001,
+                        help="Time step of simulation (s)")
+    parser.add_argument('--start-time', type=float, default=0.0,
+                        help="Start time of simulation (s)")
+    parser.add_argument('--end-time', type=float, default=0.05,
+                        help="End time of simulation (s)")
+    parser.add_argument('--dvp', type=str, default="v",
+                        help="Quantity to postprocess. Choose 'v' for velocity, 'd' for displacement, 'p' for pressure,"
+                             " or 'wss' for wall shear stress.")
+    parser.add_argument('--bands', default="25,100000",
+                        help="Input lower and upper band for band-pass filtered displacement, in a list of pairs. For "
+                             "example: --bands '100 150 175 200' gives you band-pass filtered visualization for the "
+                             "band between 100 and 150, and another visualization for the band between 175 and 200.")
+    parser.add_argument('--points', default="0,1",
+                        help="Input list of points")
+
+    args = parser.parse_args()
+
+    # Set default mesh path if not provided
+    args.mesh_path = args.folder / "Mesh" / "mesh.h5" if args.mesh_path is None else args.mesh_path
+
+    return args
+
+
+def get_coords(mesh_path: Union[str, Path]) -> np.ndarray:
+    """
+    Get coordinates from a mesh file.
+
+    Args:
+        mesh_path (Union[str, Path]): Path to the mesh file.
+
+    Returns:
+        np.ndarray: Array containing the coordinates.
+    """
+    mesh = h5py.File(mesh_path, "r")
+    coords = mesh['mesh/coordinates'][:, :]
+    return coords
+
+
+def get_surface_topology_coords(out_file: Union[str, Path]) -> tuple:
+    """
+    Get surface topology and coordinates from an output file.
+
+    Args:
+        out_file (Union[str, Path]): Path to the output file.
+
+    Returns:
+        tuple: Tuple containing the surface topology and coordinates.
+    """
+    mesh = h5py.File(out_file, "r")
+    topology = mesh["Mesh/0/mesh/topology"][:, :]
+    coords = mesh["Mesh/0/mesh/geometry"][:, :]
+    return topology, coords
+
+
+def get_domain_topology(mesh_path: Union[str, Path]) -> Tuple[np.ndarray, np.ndarray, np.ndarray]:
```

Comment:
Please see the comment below. This function does not work with multiple fluid domain and also assumes fluid domain ID to be 1 and solid domain to be 2 or larger than that, which can be dangerous.